### PR TITLE
assignment 2 pr

### PR DESCRIPTION
**Pull request by zemra-labs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,15 +109,15 @@ People
...
 * [Yuxue Mei](people/yuxue-mei.md) - ym434 - [http://ec2-yuxue2017.com.s3-website.us-east-2.amazonaws.com/grading.html](http://ec2-yuxue2017.com.s3-website.us-east-2.amazonaws.com/grading.html)[![CircleCI](https://circleci.com/gh/Detective2016/skeleton.svg?style=svg)](https://circleci.com/gh/Detective2016/skeleton)
-* [Andi Mitre](people/andi-mitre.md) - am2886 - [http://startup-sys-andi.s3-website-us-east-1.amazonaws.com/grading.html](http://startup-sys-andi.s3-website-us-east-1.amazonaws.com/grading.html)
-* [Yian Mo](people/yian-mo.md) - ym443 - [http://yian.ml.s3-website.us-east-2.amazonaws.com/grading.html](http://yian.ml.s3-website.us-east-2.amazonaws.com/grading.html)[![CircleCI](https://circleci.com/gh/moyk/skeleton.svg?style=svg)](https://circleci.com/gh/moyk/skeleton) 
+* [Andi Mitre](people/andi-mitre.md) - am2886 - [http://ec2-54-234-72-129.compute-1.amazonaws.com](http://ec2-54-234-72-129.compute-1.amazonaws.com) [![CircleCI](https://circleci.com/gh/andimitre/skeleton.svg?style=svg)](https://circleci.com/gh/andimitre/skeleton)
```
Comment (amfleming) [inline README.md:119]: Whoops!

**Review by amfleming — CHANGES_REQUESTED** [review]
Only change your own line

File: README.md (hunk 1 above)
Comment (zemra-labs) [reply README.md:119]: @amfleming I only see a space difference on the one below line 120?

Comment (zemra-labs) [reply README.md:119]: @amfleming updated this PR without spacing issues here 
https://github.com/CT-CS5356-Fall2017/cs5356/pull/388

**Pull request closed without merging**